internal/test/builder/builders.go

```diff
@@ -316,6 +316,7 @@ type ClusterClassBuilder struct {
 	controlPlaneNodeDeletionTimeout           *metav1.Duration
 	machineDeploymentClasses                  []clusterv1.MachineDeploymentClass
 	machinePoolClasses                        []clusterv1.MachinePoolClass
+	namingStrategy                            *clusterv1.ControlPlaneClassNamingStrategy
```

Comment:
```suggestion
	controlPlaneNamingStrategy                            *clusterv1.ControlPlaneClassNamingStrategy
```
\+ move it up to the others